TOC.md

```diff
@@ -311,6 +311,9 @@
     - [报警规则](/reference/tiflash/alert-rules.md)
     - [性能调优](/reference/tiflash/tune-performance.md)
     - [FAQ](/reference/tiflash/faq.md)
+  + Titan
```

Comment:
@kissmydb PTAL whether this is appropriate, thanks!

Reply:
LGTM